packages/frontend/src/hooks/useColumnTotals.ts

```diff
@@ -18,20 +21,24 @@ type Args = {
     itemsMap: Record<FieldId, Field | TableCalculation>;
 };
 
-export const isSummable = (item: Field | TableCalculation) => {
-    if (isField(item) || isAdditionalMetric(item)) {
+export const isSummable = (item: Item) => {
+    if (isTableCalculation(item)) {
+        return false;
+    }
+
+    if (isField(item) || isMetric(item) || isAdditionalMetric(item)) {
         const numericTypes: string[] = [
             DimensionType.NUMBER,
             MetricType.NUMBER,
             MetricType.COUNT,
-            MetricType.COUNT_DISTINCT,
             MetricType.SUM,
         ];
         const isPercent = item.format === 'percent';
         const isDatePart = isDimension(item) && item.timeInterval;
         return numericTypes.includes(item.type) && !isPercent && !isDatePart;
     }
-    return true;
+
+    return false;
```

Comment:
what cases are caught by this?

Reply:
none, just makes the return type explicit

Reply:
shall we leave it and not change it in that case?